aspnetcore/blazor/tooling.md

```diff
@@ -675,20 +675,39 @@ For more information on template options, see the following resources:
 
 ## .NET WebAssembly build tools
 
-:::moniker-end
-
-:::moniker range=">= aspnetcore-7.0"
-
 The .NET WebAssembly build tools are based on [Emscripten](https://emscripten.org/), a compiler toolchain for the web platform. To install the build tools, use ***either*** of the following approaches:
 
 * For the **ASP.NET and web development** workload in the Visual Studio installer, select the **.NET WebAssembly build tools** option from the list of optional components.
 * Execute `dotnet workload install wasm-tools` in an administrative command shell.
 
+> [!NOTE]
+> .NET WebAssembly build tools for .NET 6 projects
+>
+> The `wasm-tools` workload installs the build tools for .NET 7 projects. However, the .NET 7 version of the build tools are incompatible with existing projects built with .NET 6. Projects using the build tools that must support both .NET 6 and .NET 7 must use multi-targeting.
+>
+> Use the `wasm-tools-net6` workload for .NET 6 projects when developing apps with the .NET 7 SDK. To install the `wasm-tools-net6` workload, execute the following command from an administrative command shell:
```

Comment:
There is already `wasm-tools-net7` and `wasm-tools` is Net8 I think

Reply:
I made an update for `wasm-tools` to say it applies to the latest release, but I don't know what this should say for .NET 6 projects. You may need to supply the exact line of text that makes sense.

You probably already figured out ..........

![mccoy](https://github.com/dotnet/AspNetCore.Docs/assets/1622880/4025a3d6-2e0c-4688-90d5-18e27f1d27f8)

### *Damn it Jim! I'm a doc author, not a compiler expert!*

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&mdash; Dr. Leonard H. "Bones" McCoy ([DeForest Kelley](https://www.imdb.com/name/nm0001420/)) [Star Trek](http://www.startrek.com/), ©1966-69 [Paramount Pictures](http://www.paramount.com)

# 🙈😆

Reply:
Note in passing that the rest of the AOT content (and runtime relinking in a following section) is at ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-7.0#ahead-of-time-aot-compilation

I think a long-range task is to combine the content from both docs into one new article in the *Fundamentals* node. I opened an issue to do the work at ...

**Combine and relocate WASM build tools content**
https://github.com/dotnet/AspNetCore.Docs/issues/30153

With the .NET 8 workload, which will go right into 24Q1, I won't reach that for a while tho 🏃.